### ISSUE TITLE
Issue with OpenGL renderer 0.9.5

### ISSUE THREAD

**Issue by Sheiky911** [melonDS-emu/melonDS#1678]
I have an issue as soon as I choose OpenGL for rendering, nothing works on both screens. 

Have you guys ever heard of such an issue ? 

It's on all games that I've tried so far, here's gameplay on Dragon Quest 1R2

Help is much appreciated

https://user-images.githubusercontent.com/132483500/235980404-08718f52-f3c7-4bcd-bce0-7e99608d80b4.mp4

I am running an AMD 6700 XT with last drivers

**Comment by RSDuck** [comment]
get your GPU manufacturer to release a fix.

**Comment by TsukiZero** [comment]
This happens with some games, IIRC. Dunno on the cause, but it just happens.

Check if it happens with DeSmuME X432R too

**Comment by FleetingFlame** [comment]
I have the same issue with 3d rendered scenes in Pokémon games, the only "fix" I know is to swap the 3d renderer to software rather than OpenGL

**Comment by FleetingFlame** [comment]
> 

Yep, an RX 6900 XT! I noticed the glitch introduced with the OpenGl driver too, what a shame

**Comment by TsukiZero** [comment]
> > I have the same issue with 3d rendered scenes in Pokémon games, the only "fix" I know is to swap the 3d renderer to software rather than OpenGL
> 
> Do you have an AMD GPU? If so then it's a driver issue, ever since AMD "optimized" their OpenGL driver it caused issues with all kinds of software.

It also happens to a lesser degree in NFS Carbon on an Nvidia GeForce.
